app/services/employee_finder.rb

```diff
@@ -0,0 +1,23 @@
+require "slack-ruby-client"
+
+class EmployeeFinder
+  def initialize
+    configure_slack
+  end
+
+  def employee_exists?(slack_username)
```

Comment:
I'm definitely a fan of consistency, but also explicit naming conventions and the purpose of this method isn't to process anything but instead to find something.  If these methods are to be treated as the main entry points for the service classes regardless of what they do, then perhaps a more generic name might be better?